src/frontend/komponenter/Fagsak/Dokumentutsending/DokumentutsendingSkjema.tsx

```diff
@@ -144,6 +144,15 @@ const DokumentutsendingSkjema: React.FC<Props> = ({ bruker }) => {
         }
     };
 
+    function hentTittelForBarnIBrevSkjema(årsak: DokumentÅrsak) {
+        if (barnBosattMedSøkerÅrsaker.includes(årsak)) {
+            return 'Hvilke barn er bosatt med søker?';
+        } else if (barnSøktForÅrsaker.includes(årsak)) {
+            return 'Hvilke barn er søkt for?';
+        }
+        throw new Error('Må definere hva slags tittel årsak skal');
```

Comment:
Burde vi gi en tydeligere feilmelding om hva som er galt? Typ `Ukjent tittel for valg av barn i brev` eller noe i den duren

Reply:
Eller litt usikker på om vi egentlig trenger å kaste feil 🤔